src/components/awesome_link.cr

```diff
@@ -1,14 +1,21 @@
 class AwesomeLink < BaseComponent
+  enum Tags
+    Crypto
```

Comment:
Seems strange to me that we only have two tags. I get why we're tagging Adult, but does Crypto need to be treated specially also? If not should we also have tags for everything else?

Reply:
I added the `Crypto` only for the purposes of being able to test a multiple tagging system. I actually had an idea for tags in the shards section where shards that are made specifically for Lucky could be tagged as such. Then there's other shards that work with Lucky, but they're not necessarily "lucky" shards (e.g. multi_auth, sitemapper, etc...). I could drop it for now, and then we can talk about what would make most sense in a separate PR.